### PR TITLE
Fix: Empty task list, Fixes #14

### LINKED ISSUE #14: Empty task list
Would love to use it, but:

image

I have 22 tasks for today.

### PR DESCRIPTION
**Pull request by ronilaukkarinen**
This makes the filters show up again. 👍 
I also added a CHANGELOG and debug logging in dev.

### CODE REVIEW

File: src/pages/secure/index.vue [hunk 1]
```diff
@@ -103,16 +103,49 @@ const { data: fetchedTasks, pending: taskPending } = useFetch<Task[]>("https://a
   query: {
     filter: fetchFilter,
   },
+  onResponse({ response }) {
+    if (process.dev) {
```
Comment (selfire1) [inline src/pages/secure/index.vue:107]:
```suggestion
    if (import.meta.dev) {
```

File: src/pages/secure/index.vue [hunk 2]
```diff
@@ -103,16 +103,49 @@ const { data: fetchedTasks, pending: taskPending } = useFetch<Task[]>("https://a
...
+        taskCount: response._data?.length,
+        firstTask: response._data?.[0]
+      })
+    }
+  }
 })
 
 const { customSort } = useTasks();
 const tasks = computed(() => {
+  if (process.dev) {
```
Comment (selfire1) [inline src/pages/secure/index.vue:119]:
```suggestion
  if (import.meta.dev) {
```

File: src/pages/secure/index.vue [hunk 3]
```diff
@@ -103,16 +103,49 @@ const { data: fetchedTasks, pending: taskPending } = useFetch<Task[]>("https://a
...
+  }
+
   if (!sortQuery.value) {
     return fetchedTasks.value;
   }
-  return customSort(fetchedTasks.value, sortQuery.value);
+  return customSort(fetchedTasks.value || [], sortQuery.value);
 });
 
+if (process.dev) {
```
Comment (selfire1) [inline src/pages/secure/index.vue:133]:
```suggestion
if (import.meta.dev) {
```

**Review by selfire1 — CHANGES_REQUESTED** [review]
Thank you!

Only change request is to use `[import.meta.dev](https://nuxt.com/docs/api/advanced/import-meta#runtime-app-properties)`. It is the [recommended method](https://github.com/nuxt/nuxt/discussions/18779#discussioncomment-8864742) to check if Nuxt is in development mode.

**Comment by ronilaukkarinen** [comment]
OK, this whole stack and Nuxt is very new to me 😅 I'll check it out and fix soon.

**Review by selfire1 — APPROVED** [review]

**Pull request merged**